### PR TITLE
Uploaded new UserPrivateSmall icon

### PR DESCRIPTION
**Pull request by Zohar-Telor**

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**
Commit: **fbc8233d97d9f921bed04139b6eea8892afc59a3**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2229/

File: packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts [hunk 1]
```diff
@@ -3275,6 +3275,15 @@ export const generalIconsMetadata: Array<IconMetadata> = [
       "24": "Github"
     }
   },
+  {
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts:3278]: the small suffix should be added just for the size map itself.
here's an example to better emphasis what i'm referring to:
```js
 {
    "title": "Cloud",
    "category": "Files & Data",
    "description": "",
    "tags": [],
    "sizes": {
      "18": "CloudSmall",
      "24": "Cloud"
    }
  },
```

File: packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts [hunk 2]
```diff
@@ -3275,6 +3275,15 @@ export const generalIconsMetadata: Array<IconMetadata> = [
...
+    "title": "UserPrivateSmall",
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts:3279]:
```suggestion
    "title": "UserPrivate",
```

File: packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts [hunk 3]
```diff
@@ -3275,6 +3275,15 @@ export const generalIconsMetadata: Array<IconMetadata> = [
...
+    "category": "User",
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts:3280]:
```suggestion
    "category": "Users",
```

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
added some comments to fix

File: packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts (hunk 2 above)
Comment (Zohar-Telor) [reply packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts]: @TalZa1810 Yes i've noticed that and fixed.

File: packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts (hunk 3 above)
Comment (Zohar-Telor) [reply packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts]: Our category is called 'User' not 'Users' so...
image

Comment (TalZa1810) [reply packages/wix-ui-icons-common/src/classic-editor/general/metadata.ts]: seems like there's a difference bewteen wsr and base-ui. in wsr it's [Users](https://www.wix-style-react.com/?path=/story/foundations-icons--icons). we need to align this but let's leave this as is for now.

**Review by TalZa1810 — APPROVED** [review]

**Pull request merged**